main-packages-list.txt

```diff
@@ -48,7 +48,7 @@ python3-pyghmi >= 1.5.14-2.1.el8ost
 python3-tooz >= 2.8.0-0.20210324235001.54448e9.el8
 python3-scciclient >= 0.9.1-0.20210720102209.34ccd96.el8
 python3-stevedore >= 3.3.0-0.20210325001012.7d7154f.el8
-python3-sushy >= 3.12.1-0.20211122142104.806622c.el8
+python3-sushy >= python-sushy-3.12.1-0.20220512112437.2b2417e.el8
```

Comment:
Do we really need to duplicate the package name?

Reply:
should just be the version